### DISCUSSION TITLE
react-icons go below the text instead of next to the text

### DISCUSSION THREAD

**Discussion by gerardofisch** [tailwindlabs/tailwindcss#10061] [Help] [answered]
**What version of Tailwind CSS are you using?**
v3.2.4

**What build tool (or framework if it abstracts the build tool) are you using?**
React js 18.20

**What version of Node.js are you using?**
v19.2.0

**What browser are you using?**
Firefox

**What operating system are you using?**
Ubuntu 22.04.1

**Code:**
```
import {FcHeadset} from "react-icons/fc" 
  return (
    <div>
      <p>
        This icon goes below: <FcHeadset />
      </p>
    </div>
  );
  ```
  My problem is that the icon goes below the text intead of next to the text

**Comment by reinink** [comment, marked as answer]
Hey try this!

```jsx
import { FcHeadset } from "react-icons/fc";

return (
  <div>
    <p className="flex items-center">
      <span className="mr-2">Text on the left</span>
      <FcHeadset />
    </p>
  </div>
);
```

Here is an example illustrating this approach on Tailwind Play:

https://play.tailwindcss.com/EwGEUiyOsH

Hope that helps! 💪

**Reply by gerardofisch** [reply]
It worked.
Thank you!!
